client/src/Main.elm

```diff
@@ -30,7 +32,7 @@ import Task
 import TypedSvg exposing (g, svg, text_)
 import TypedSvg.Attributes exposing (dy, stroke, textAnchor, transform, viewBox)
 import TypedSvg.Core exposing (Svg)
-import TypedSvg.Types exposing (AnchorAlignment(..), Paint(..), Transform(..), em)
+import TypedSvg.Types exposing (AnchorAlignment(..), Paint(..), Transform(..), YesNo, em)
```

Comment:
You're not using SVG. Remove

Reply:
I use Svg for the visual representation.
If I remove one of these import, the code won't compile.

Reply:
[This](https://github.com/vibbits/GGW-Elm/pull/18#discussion_r795492666) seems to be the only place you're using it?